discovery-provider/src/queries/queries.py

```diff
@@ -33,7 +33,8 @@ def get_users():
     with db.scoped_session() as session:
         # Create initial query
         base_query = session.query(User)
-        base_query = base_query.filter(User.is_current == True, User.is_ready == True)
+        # Don't return the user if they have no wallet or handle (user creation did not finish properly on chain)
+        base_query = base_query.filter(User.is_current == True, User.wallet != None, User.handle != None)
```

Comment:
@raymondjacobson why not just change the is_ready condition?

Reply:
like in src/tasks/index/users.py, just remove name from the is_ready initial condition

Reply:
@hareeshnagaraj changing the condition would not change data retroactively. i think a better analogue would be deleting "is_ready" from the db totally and removing all references

Reply:
changing the is_ready condition is only a solution for *new* accounts. and i also wanted to make this as least intrusive as possible

Reply:
this is to unbrick users who have is_ready already committed incorrectly